XenModel/Actions/Updates/DownloadAndUpdateClientAction.cs

```diff
@@ -102,6 +105,13 @@ private void DownloadFile()
 
                     //start the download
                     _updateDownloadState = DownloadState.InProgress;
+
+                    if (!string.IsNullOrEmpty(_authToken))
+                    {
+                        NameValueCollection myQueryStringCollection = new NameValueCollection();
+                        myQueryStringCollection.Add("burritoToken", _authToken);

```

Comment:
Please consider naming it using the `INTERNAL_STAGE_AUTH_TOKEN` constant (i.e. `"InternalStageAuthToken"`) instead of mentioning the code name of burrito.

Reply:
surething